src/ImageSharp.Drawing/Brushes/PatternBrush{TColor}.cs

```diff
@@ -156,6 +143,38 @@ public override void Dispose()
             {
                 // noop
             }
+
+            /// <inheritdoc />
+            internal override void Apply(float[] scanlineBuffer, int scanlineWidth, int offset, int x, int y)
```

Comment:
Ok, one change serious thing: `PatternBrushAppicator.Apply()` seems to be uncovered!